### ISSUE TITLE
Would it be feasible to add class in table td?

### ISSUE THREAD

**Issue by olgabrani** [apyrgio/synnefo#223]
In some columns of dataTables, the value is `True/False`.
I wonder if we could easily wrap this value in a span and give it a class, so that we can style  True (or False).
I think at the moment it is not very easy for one to have a spherical view of the True/False items at a glance.

**Comment by athinab** [comment]
You are right. It would be a visual help to add some style to True/False values. And it's doable.
My only concern is how much time will it cost when the table gets thousands of data (use of "select all" button). 

But I think we can try it and see.
